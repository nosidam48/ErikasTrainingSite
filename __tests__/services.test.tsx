import React from 'react'
import renderer from 'react-test-renderer'
import Services from '../pages/services/index'

it('renders about unchanged', () => {
  const tree = renderer.create(<Services />).toJSON()
  expect(tree).toMatchSnapshot()
})