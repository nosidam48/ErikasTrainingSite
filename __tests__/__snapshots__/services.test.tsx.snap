// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders about unchanged 1`] = `
<div
  className="services"
>
  <div
    className="serviceItem"
  >
    <h3>
      Single Session
    </h3>
    <div
      className="serviceImage"
    >
      <img
        alt=""
        height="134"
        priority={true}
        src="/images/dogBowl.png"
        width="177"
      />
    </div>
    <p
      className="sessionText"
    >
      Single Sessions can be booked for adult dogs or puppies. The first session is a 90-minute consultation to assess your dog’s routine and needs. Erika will put together a training plan tailored to your goals that will get you training right away! Follow-up sessions are one hour long and can be scheduled as needed.
    </p>
    <strong
      className="sessionPrice"
    >
      First Session: $100
    </strong>
    <strong>
      Follow ups: $90
    </strong>
    <button
      className="button1"
      href="/contact"
    >
      Contact Erika
    </button>
  </div>
  <div
    className="serviceItem"
  >
    <h3>
      Training Package
    </h3>
    <div
      className="serviceImage"
    >
      <img
        alt=""
        height="145"
        priority={true}
        src="/images/dogFace.png"
        width="175"
      />
    </div>
    <p
      className="packageText"
    >
      Erika’s Training Package can be booked for adult dogs or puppies. It comes with three hour-long sessions for those pet parents who want continued support for their training efforts. This option is great for addressing challenging behavioral issues or just getting some extra support with the basics. Erika will update your training plan after each session to keep you and your dog on track!
    </p>
    <strong>
      Three Pack: $240
    </strong>
    <button
      className="button2"
      href="/contact"
    >
      Contact Erika
    </button>
  </div>
  <div
    className="serviceItem"
  >
    <h3>
      Complete Puppy Session
    </h3>
    <div
      className="serviceImageLarge"
    >
      <img
        alt=""
        height="154"
        priority={true}
        src="/images/pawHouse.png"
        width="136"
      />
    </div>
    <p>
      The Complete Puppy Session includes a two-hour crash course on all things puppy. This option is great for new puppy parents, especially those with busy schedules. Get all of your questions answered and learn new techniques to get your puppy started off on the right paw!
    </p>
    <p>
      Topics covered include:
    </p>
    <div
      className="topicList"
    >
      <div>
        Biting
      </div>
      <div>
        Potty Training
      </div>
      <div>
        Jumping
      </div>
      <div>
        Leash Manners
      </div>
      <div>
        Crate Training
      </div>
      <div>
        And More...
      </div>
    </div>
    <strong
      className="puppySession"
    >
      Complete Puppy Session: $175
    </strong>
    <button
      className="button3"
      href="/contact"
    >
      Contact Erika
    </button>
  </div>
</div>
`;
